plugins/source/facebookmarketing/resources/plugin/plugin.go

```diff
@@ -0,0 +1,49 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/facebookmarketing/client"
+	"github.com/cloudquery/cloudquery/plugins/source/facebookmarketing/resources/services"
+	"github.com/cloudquery/plugin-sdk/plugins/source"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+var (
+	Version = "development"
+)
+
+func Plugin() *source.Plugin {
+	return source.NewPlugin(
+		"cloudquery-facebookmarketing",
```

Comment:
```suggestion
		"facebookmarketing",
```

I don't think we prefix our own plugins